numpy-stubs/__init__.pyi

```diff
@@ -58,6 +58,7 @@ _DType2 = TypeVar("_DType2", bool_, float32, float64, int8, int16, int32, int64,
 _DTypeObj = TypeVar("_DTypeObj", bound=Union[dtype, int, float])
 _ShapeType = Union[int, Tuple[int, ...], List[int]]
 _AxesType = Union[int, Tuple[int, ...], List[int]]
+_InterpolationType = Literal['linear', 'lower', 'higher', 'midpoint', 'nearest']
```

Comment:
I added this as it's own type rather than have the literal in the function signature, but it's only used in the `percentile` and `quantile` functions (and I haven't even typed quantile yet), so I completely understand if you would prefer this in the function signature.

Reply:
No, it's fine. I like defining this elsewhere. Keeps the function signature short.